src/library/traktor/traktorfeature.cpp

```diff
@@ -216,7 +216,7 @@ TreeItem* TraktorFeature::importLibrary(const QString& file) {
 
     //Parse Trakor XML file using SAX (for performance)
     QFile traktor_file(file);
-    if (!traktor_file.open(QIODevice::ReadOnly | QIODevice::Text)) {
+    if (!Sandbox::askForAccess(file) && !traktor_file.open(QIODevice::ReadOnly | QIODevice::Text)) {
```

Comment:
still wrong

Reply:
Thank you for double checking. Fixed.

Reply:
Unrelated: QIODevice::Text seems to be wrong here. The XML parser expects binary data. Treating XML as text is not good idea.